### ISSUE TITLE
Error when exporting phenopacket

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#3950]
```
2023-05-25 11:00:58,159 - [scout.server.app](http://scout.server.app/) - ERROR: Exception on /cust139/Hypophosphatemia-Family1/phenotype_export [POST] [in /venv/lib/python3.8/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/venv/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/venv/lib/python3.8/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/home/worker/app/scout/server/blueprints/cases/views.py", line 381, in phenotype_export
    phenopacket_json = phenopacketapi.phenopacket_from_case(case_obj)
  File "/home/worker/app/scout/server/extensions/phenopacket_extension.py", line 59, in phenopacket_from_case
    for term in phenotype_terms:
TypeError: 'NoneType' object is not iterable
```

**Comment by northwestwitch** [comment]
@dnil I can work on fixing this if you haven't started yet!

**Comment by dnil** [comment]
Nono, my bad, I'll fix the empty field export. 😸

**Comment by dnil** [comment]
Heh, it's fine, but thanks - I've fixed!

**Comment by dnil** [comment]
Closed by #3951
